lib/stats/Makefile.am

```diff
@@ -8,6 +8,7 @@ statsinclude_HEADERS = \
 	lib/stats/stats-counter.h		\
 	lib/stats/stats-cluster.h		\
 	lib/stats/stats-aggregator.h	\
+	lib/stats/stats-change-per-second.h \
```

Comment:
Can we create a new folder for all aggregator-related units (for example, `lib/stats/aggregator`)?
It would be less confusing in a separate folder, far away from the other stats-related units.